### PR TITLE
Add ui docs description

### PR DESCRIPTION
**Pull request by Aoao54**
Add ui docs description

### CODE REVIEW

**Review by yujunz — APPROVED** [review]

File: docs/features/temporal-ui.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 This page is WIP. Feel free to contribute [on github](https://github.com/alexandrevilain/temporal-operator/edit/main/docs/features/temporal-ui.md).
 
+Temporal-operator supports configuring WEB UI for Temporal clusters. You can get more informations about Event histories backup on [Temporal documentation](https://docs.temporal.io/web-ui).
```
Comment (alexandrevilain) [inline docs/features/temporal-ui.md:5]:
```suggestion
Temporal-operator supports configuring web UI for Temporal clusters. You can get more informations about web UI on [Temporal documentation](https://docs.temporal.io/web-ui).
```

File: docs/features/temporal-ui.md [hunk 2]
```diff
@@ -26,6 +28,8 @@ spec:
 
 ## Create Ingress
 
+Ingress is an optional ingress configuration for the UI. If lived empty, no ingress configuration will be created and the UI will only by available trough ClusterIP service.
```
Comment (alexandrevilain) [inline docs/features/temporal-ui.md:31]:
```suggestion
Ingress is an optional ingress configuration for the UI. If leaved empty, no ingress configuration will be created and the UI will only by available through ClusterIP service.
```

File: docs/features/temporal-ui.md [hunk 3]
```diff
@@ -77,6 +81,8 @@ spec:
 
 ## Override UI deployment
 
+Override UI deployment can be used to set [Web UI env vars](https://docs.temporal.io/references/web-ui-environment-variables).
```
Comment (alexandrevilain) [inline docs/features/temporal-ui.md:84]:
```suggestion
Web UI overrides can be used to set [web UI environment variables](https://docs.temporal.io/references/web-ui-environment-variables).
```

**Review by alexandrevilain — COMMENTED** [review]
Thanks for this useful contribution !! 
I added some suggestions :)

**Comment by yujunz** [comment]
Just realize the description should be "Add ui docs ~does~ description".

**Review by yujunz — CHANGES_REQUESTED** [review]
Minor comments on commit message.

**Review by alexandrevilain — APPROVED** [review]
Thanks !!

**Pull request merged**